filesystem/iso9660.ksy

```diff
@@ -7,206 +7,84 @@ meta:
     pronom: fmt/468
     wikidata: Q815645
   license: CC0-1.0
+  endian: be
+  encoding: ASCII
 doc: |
   ISO9660 is standard filesystem used on read-only optical discs
   (mostly CD-ROM). The standard was based on earlier High Sierra
   Format (HSF), proposed for CD-ROMs in 1985, and, after several
-  revisions, it was accepted as ISO9960:1998.
-
+  revisions, it was accepted as ISO9660:1998.
   The format emphasizes portability (thus having pretty minimal
   features and very conservative file names standards) and sequential
   access (which favors disc devices with relatively slow rotation
   speed).
+doc-ref:
+  - ecma-119 https://www.ecma-international.org/publications/files/ECMA-ST/Ecma-119.pdf
```

Comment:
This URL is now keeps redirecting (with status code [`301 Moved Permanently`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301)), so please link directly to the target.

Reply:
fixed